Survey/Survey/Sources/Presentation/Modules/Login/LoginViewModel.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  LoginViewModel.swift
+//  Survey
+//
+//  Created by Taher on 7/11/22.
+//
+
+import Combine
+import SwiftUI
+
+final class LoginViewModel: ObservableObject {
+
+    // MARK: Input
+
+    @Published var email: String = .empty
+    @Published var password: String = .empty
+    var didTapLoginButton = PassthroughSubject<Void, Never>()
+
+    // MARK: Output
+
+    @Published var errorMessage: String = .empty
+
+    private let loginUseCase: LoginUseCaseProtocol
+    private var disposables = Set<AnyCancellable>()
```

Comment:
As [this discussion](https://github.com/Shayokh144/ic-survey-swiftui-combine/pull/41#discussion_r1016102305), I think `cancellables` is a better name 🙏